src/commonMain/kotlin/wizard/Dependencies.kt

```diff
@@ -201,4 +201,19 @@ val SQLDelightDriverNative = SQLDelightPlugin.copy(
 val SQLDelightDriverJs = SQLDelightPlugin.copy(
     id = "sqljs-driver",
     platforms = setOf(ComposePlatform.Browser)
+)
+
+val ApolloPlugin = Dependency(
+    title = "ApolloGraph",
```

Comment:
"Apollo Kotlin" is a better name, isn't it?
https://www.apollographql.com/docs/kotlin

Reply:
It 100% is 👍